lib/render_async/view_helper.rb

```diff
@@ -44,7 +45,8 @@ def render_async(path, options = {}, &placeholder)
                                           error_message: error_message,
                                           error_event_name: error_event_name,
                                           retry_count: retry_count,
-                                          interval: interval
+                                          interval: interval,
+                                          lazy_load: lazy_load
```

Comment:
Also, could you fix the RSpec tests failing because of this added option?

Reply:
Yeah, lot of them were breaking and I wanted to make sure if team is fine with this approach before fixing them. Will work on them now.